examples/private/main.tf

```diff
@@ -0,0 +1,224 @@
+resource "aws_security_group" "client_sec_group" {
+  name        = "${var.deployment_prefix}-client-sg"
+  description = "client security group"
+  vpc_id      = var.vpc_id
+  tags        = var.tags
+
+  ingress {
+    description = "Allow oubound to ssh"
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
```

Comment:
From https://registry.terraform.io/providers/hashicorp/aws/3.42.0/docs/resources/security_group_rule

> Setting protocol = "all" or protocol = -1 with from_port and to_port will result in the EC2 API creating a security group rule with all ports open. This API behavior cannot be controlled by Terraform and may generate warnings in the future.

Reply:
Yes, that is intended for the client SG. I may harden it in future but I wanted it to be very accessible for testing. The brokers are all well protected.